docs/networking.md

```diff
@@ -0,0 +1,262 @@
+
+
+
+<!-- - Objectives of the document
+- Importance of networking in cloud development and operations -->
+
+## Network architecture
+
+The ASEA network centers around a Transit Gateway, efficiently directing traffic between the internet, and AWS services in various different AWS Accounts on the platform. Networking is broken up into two separate accounts: 
+
+- The Perimeter Account houses our Checkpoint Firewalls which monitor and restrict traffic in and out of the platform. 
+- The Shared Network Account houses the Transit Gateway and acts as a central hub. this account houses all of the VPCs (except for the Perimeter VPC in the Perimeter account) and most other networking resources. These are made available via AWS Resource Access Manager (RAM) to the appropriate accounts based on their OU in the organization. Security Groups are then put in place restrict traffic between resources deployed in the VPCs. All this eliminates the need to create duplicate resources in multiple accounts, reducing the operational overhead of managing those resources in every single account.
+
+The reason for the separation between the Shared Networking and Perimeter accounts is to facilitate networking and security "separation of duties". In summary, the ASEA's networking architecture ensures centralized, organized and secure communication through Transit Gateway routing, separated Perimeter VPC security, and Shared Network resource management as shown below in BC Gov ASEA's complete networking diagram:
+
+![networking-architecture](images/networking/network-architecture.png)
+
+For further reading beyond this document please visit the [AESA network architecture docs](https://aws-samples.github.io/aws-secure-environment-accelerator/latest/architectures/sensitive/network/). Note that we use the [GWLB architecture](https://aws-samples.github.io/aws-secure-environment-accelerator/latest/architectures/sensitive/diagrams/#14-additional-perimeter-patterns) with third-party firewalls.
+
+
+
+<!-- - High-level overview of the ASEA network architecture
+- network architecture diagram is available in sharepoint
+- link to the AESA network architecture docs (We use the GWLB architecture with third-party firewalls) -->
+
+## Gateway load balancers and CheckPoint firewalls
+
+In the Perimeter account we use a [Gateway Load Balancer (GWLB)](https://aws.amazon.com/elasticloadbalancing/gateway-load-balancer/) to distribute traffic load between our firewall instances. These run on EC2, in a highly available pair. We use Checkpoint Firewalls from the AWS Marketplace, alongside the Checkpoint Firewall Manager where we use Checkpoint's Smart Console to set up traffic rules. Only HTTP/HTTPS traffic is allowed through the firewalls, all other traffic is blocked. This includes SSH egress traffic which can cause issues with accessing git repositories and other services that use SSH. To access git repositories, use HTTPS instead of SSH.
+
```

Comment:
Add some direction that if non https egress is required then reach out to the Public Cloud team for assistance